packages/cli/src/bin/store.ts

```diff
@@ -0,0 +1,67 @@
+import { Dataset, RunCompletion } from "@empiricalrun/types";
+import { promises as fs } from "fs";
+
+const cwd = process.cwd();
+const datasetFileName = "dataset.json";
+const cacheDir = ".empiricalrun";
+const outputFilePath = (datasetId: string, runId: string) =>
+  `${cwd}/${cacheDir}/${datasetId}/${runId}.json`;
+const datasetFilePath = `${cwd}/${cacheDir}/${datasetFileName}`;
+
+export class RunsFileStore {
```

Comment:
- RunsStore abstraction
- internally change storage from file / database
   - duckdb in the middle for the file storage
- what json format works better for duckdb - single vs multiple files
- save it also via duckdb? avoid fs APIs